cognite/client/data_classes/sequences.py

```diff
@@ -229,18 +232,32 @@ def __eq__(self, other):
             and self.values == other.values
         )
 
-    def __getitem__(self, item) -> List[Union[int, str, float]]:
+    def __getitem__(self, item: int) -> List[Union[int, str, float]]:
         # slow, should be replaced by dict cache if it sees more than incidental use
         return self.values[self.row_numbers.index(item)]
 
-    def __getattr__(self, item) -> List[Union[int, str, float]]:
-        """Get a column by externalId
+    def get_column(self, external_id: str = None, id: int = None) -> List[Union[int, str, float]]:
```

Comment:
Do you have a unit test for this one?

Reply:
it gets tested in the integration tests

Reply:
That is not the purpose of the integration tests. It's ok to use it in the integration test, but it should have its own unit test, which fails only if this method breaks.

Reply:
raising a valueerror now, we could potentially support step!=1 slices in the future, but it's a bit complicated since the row numbers could be all over the place

Reply:
now is also test in unit test